ProjectWithUMLDiagram/src/com/beyond/person/core/service/implementation/DancerImpl.java

```diff
@@ -0,0 +1,57 @@
+package com.beyond.person.core.service.implementation;
+
+import com.beyond.person.core.enums.States;
+import com.beyond.person.core.exceptions.LimitException;
+import com.beyond.person.core.model.Dancer;
+import com.beyond.person.core.service.interfaces.DancerAction;
+
+public class DancerImpl implements DancerAction {
```

Comment:
@AraChilingaryan change everywhere names DancerActionService and DancerActionServiceImpl applicable for all types

Reply:
done